db/seeds.rb

```diff
@@ -2,20 +2,23 @@
 
 require 'faker'
 
+puts 'Creating admin user admin@justice.gov.uk/change_me'
```

Comment:
Are these prinouts/changes related to this PR at all? might be worth doing a separate PR for them if not? :D Might be worth using the opportunity to switch the seeds to using factories.

Reply:
No not related, removed and will add in a separate PR